builder/ebs/aws.go

```diff
@@ -1,4 +1,4 @@
-package amazon_acc
```

Comment:
Is there a reason this needs to be in the main ebs directory? One benefit of having it separate is that it doesn't get compiled into the binary. It's not a huge gain size-wise or anything, but it does help separate test-specific stuff from run-specific stuff.

Reply:
I had to move up because of cyclic dependencies. I can try to fix it to move back, I did it as a quick solution to make the tests runnable.

Reply:
hah no worries. If it is a headache it isn't worth changing it.

Reply:
I removed the other file which will cause the actual circle dep problem. The other file was used by the old version of the provisioner acc test framework you've upgraded.